plugins/pca/tasks.py

```diff
@@ -92,14 +85,14 @@ def load_entity_points_and_idx_to_id(entity_points_url: str):
 
     # Set array with correct size
     points_cnt = len(id_to_idx)
-    dimensions = len(ent["point"])
+    dimensions = len(ent.keys()) - 2    # -2 for ID and href
```

Comment:
```suggestion
    dimensions = len(ent.keys()-{"ID", "href"})
```
Entities may not have an href attr! (but they should always have an ID)